### PR TITLE
Red Dead Redemption 2 but it's a crappy 2D space man game

### PR DESCRIPTION
**Pull request by Xantholne**
## About The Pull Request

Adds 5 Cowboy hats, Black, Brown, Pink, White, and one blacker Security hat
Adds 4 Cowboy shirts with long sleeve short sleeved versions
Adds 2 Cowboy boots, Black and brown

## Why It's Good For The Game

Some variety for more western styled characters

## Changelog
add: Cowboy hats
add: Cowboy boots
add: Cowboy shirts
/:cl:

dreamseeker_00E1Xib5pD

### CODE REVIEW

**Comment by ghost** [comment]
yeehaw

**Comment by Poojawa** [comment]
probably just do the minor recolors to the digitigrade cowboy boots and this'll be perf.

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -392,3 +392,33 @@
...
+	icon_state = "cowboyhat_white"
+	item_state= "cowboyhat_white"
+
+/obj/item/clothing/head/cowboyhat/pink
+	name = "pink cowboy hat"
+	desc = "A pink cowboy? more like cowgirl hat, just don't be a buckle bunny."
+	icon_state = "cowboyhat_pink"
+	item_state= "cowboyhat_pink"
+
+/obj/item/clothing/head/cowboyhat/sec
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/head/misc.dm:420]: Perhaps one might want armor for this...

File: code/modules/vending/clothesmate.dm [hunk 2]
```diff
@@ -134,7 +134,13 @@
 					/obj/item/clothing/head/christmashatg = 3,
 					/obj/item/clothing/shoes/winterboots/christmasbootsr = 3,
 					/obj/item/clothing/shoes/winterboots/christmasbootsg = 3,
-					/obj/item/clothing/shoes/winterboots/santaboots = 3)
+					/obj/item/clothing/shoes/winterboots/santaboots = 3
```
Comment (DeltaFire15) [inline code/modules/vending/clothesmate.dm]: You forgot a comma here, likely the reason this ain't compiling

**Review by silicons — APPROVED** [review]

File: code/modules/clothing/head/misc.dm (hunk 1 above)
Comment (Xantholne) [reply code/modules/clothing/head/misc.dm:420]: Maybe, might be more suited for a balance PR, have to check if the berets actually have any or not which then I'd go ahead for.

**Pull request merged**